src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2262,29 +2265,26 @@ namespace Akka.Dispatch
         public static void RunTask(System.Func<System.Threading.Tasks.Task> asyncAction) { }
         protected override bool TryExecuteTaskInline(System.Threading.Tasks.Task task, bool taskWasPreviouslyQueued) { }
     }
-    public class BoundedDequeBasedMailbox : Akka.Dispatch.Mailbox<Akka.Dispatch.MessageQueues.BoundedMessageQueue, Akka.Dispatch.MessageQueues.BoundedDequeMessageQueue>, Akka.Dispatch.IDequeBasedMailbox
+    public sealed class BoundedDequeBasedMailbox : Akka.Dispatch.MailboxType, Akka.Dispatch.IProducesMessageQueue<Akka.Dispatch.MessageQueues.BoundedDequeMessageQueue>
```

Comment:
All of the previous built-in `Mailbox` classes have been preserved and simply had their implemented types switched to `MailboxType` - which gives us backwards compatibility inside the reference configuration.